vendor/vendor.json

```diff
@@ -315,5 +333,5 @@
 			"revisionTime": "2018-03-28T19:50:20Z"
 		}
 	],
-	"rootPath": "github.com/dragonflyoss/Dragonfly"
+	"rootPath": "github.com/dragonflyoss/dragonfly"
```

Comment:
Why change it ? Our root path should be `github.com/dragonflyoss/Dragonfly` ,not `github.com/dragonflyoss/dragonfly`.

Reply:
OK. I roll back it.